examples/bouss/petscMPIoptions

```diff
@@ -10,33 +10,29 @@
 #    https://petsc.org/release/manualpages/PC/PCSetFromOptions/

```

Comment:
@barrysmith: Is there also a petsc webpage we should mention that discusses the `mpi_*` options?  The `mpi_ksp-*` options are not discussed on the page https://petsc.org/release/manualpages/KSP/KSPSetFromOptions, for example.